.github/workflows/publish_docker_image.yaml

```diff
@@ -0,0 +1,85 @@
+name: Docker
+
+on:
+  push:
+    branches: [ "main" ]
+    tags: [ '*.*.*' ]
```

Comment:
What on alpha/beta tags like for example `2.7.1-alpha.1`?

Reply:
They work 👍

Reply:
As we are keeping the VNG fork in-sync, users can choose to fetch their Docker images from ghcr.io/signalen/* or ghcr.io/Amsterdam/signals/*.

We can remove the manual build-and-push in Signalen/build-and-push.

Reply:
@bartjkdp Two quick questions:
- What would it take to standardize on github.com/signalen? (We may want to take that pain now and avoid having parallel infrastructure. Adding members to the Signalen org on Github is trivial.)
- What if there is no tag to match, those pushes will not be built - correct?

Reply:
@tcoenen 

- Currently the Amsterdam Jenkins pipelines are thightly coupled to github.com/Amsterdam. After migration to Azure (around October this year) we are not depending on the Jenkins pipelines any more and standardize on github.com/signalen would just require adding members to github.com/Signalen and changing workflows.
- Pull requests, commits to master and matching tags will be built. Commits to master and matching tags will also be pushed.